include/rocksdb/statistics.h

```diff
@@ -396,7 +396,7 @@ enum StatsLevel {
   // Collect all stats except the counters requiring to get time inside the
   // mutex lock.
   kExceptTimeForMutex,
-  // Collect all stats expect time inside mutex lock AND time spent on
+  // Collect all stats except time inside mutex lock AND time spent on
   // compression
   kExceptDetailedTimers,
```

Comment:
mmmmm, Are you saying that there could be someone setting stats_level_ by using the enum value ? something like stats_level_ = 0 ?
I think we should go for the cleaner solution